app/src/main/java/com/udacity/asteroidradar/BindingAdapters.kt

```diff
@@ -39,3 +41,13 @@ fun bindTextViewToDisplayVelocity(textView: TextView, number: Double) {
     val context = textView.context
     textView.text = String.format(context.getString(R.string.km_s_unit_format), number)
 }
+
+@BindingAdapter("imageUrl")
+fun bindImage(imgView: ImageView, imgUrl: String?) {
+    imgUrl?.let {
```

Comment:
This `bindImage` function should use the Picasso call that is in the `MainFragment`